.github/workflows/build.yml

```
@@ -109,18 +114,18 @@ jobs:
         id: docker-vars
         with:
           image-name: ghcr.io/${{ github.repository }}/base
-          registry-secret: ${{ secrets.GHCR_PAT }}
+          registry-secret: ${{ secrets.GITHUB_TOKEN }}
       
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
 
       - name: Login to GitHub
         if: ${{ steps.docker-vars.outputs.has-docker-secret == 'true' }}
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
```

According to the documentation of the `login-action` step, this should be
```suggestion
          username: ${{ github.actor}}
```

This would be my guess on how to fix it. The change from `GTCR_PAT` to `GITHUB_TOKEN` seems okay.
Also the up to date version of `docker/login-action` is `v3`

---

I tested with these changes and got the following error in the Build and Push section
```
 Error: buildx failed with: ERROR: failed to solve: failed to push ghcr.io/illarion-ev/illarion-server/base:pr-100: unexpected status from POST request to https://ghcr.io/v2/illarion-ev/illarion-server/base/blobs/uploads/: 403 Forbidden
 ```
 seen here https://github.com/Illarion-eV/Illarion-Server/actions/runs/7427486427/job/20213407706?pr=100
 
 Not too familiar with github or its workings, so not sure if I should click on "commit suggestion" on here to test things like this for PR reviews? As I wrote in https://github.com/Illarion-eV/Illarion-Server/pull/100 I was told early on to not directly edit things in Illarion-Content due to formatting so not sure if A. this applies to server side stuff too and B. if this applies to PR reviews like this, so I'm treading on the side of caution.